tests/Remove-DbaLinkedServer.Tests.ps1

```diff
@@ -17,56 +17,85 @@ Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     BeforeAll {
         $random = Get-Random
         $instance2 = Connect-DbaInstance -SqlInstance $script:instance2
+        $instance3 = Connect-DbaInstance -SqlInstance $script:instance3
 
-        $instance2.Query("EXEC sp_addlinkedserver @server=N'LS1_$random'")
-        $instance2.Query("EXEC sp_addlinkedserver @server=N'LS2_$random'")
-        $instance2.Query("EXEC sp_addlinkedserver @server=N'LS3_$random'")
+        $linkedServerName1 = "LS1_$random"
+        $linkedServerName2 = "LS2_$random"
+        $linkedServerName3 = "LS3_$random"
+        $linkedServerName4 = "LS4_$random"
```

Comment:
Please use `dbatoolscli` as a prefix for these objects. Majority of our test should be using this when we create objects on the instance incase a user wants to run our tests in their own environment.

Reply:
updated this as well